docs/docs/introduction.mdx

```diff
@@ -27,18 +28,19 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 Concretely, the framework consists of the following open-source libraries:
 
-- **`langchain-core`**: Base abstractions and LangChain Expression Language.
-- **`langchain-community`**: Third party integrations.
-  - Partner packages (e.g. **`langchain-openai`**, **`langchain-anthropic`**, etc.): Some integrations have been further split into their own lightweight packages that only depend on **`langchain-core`**.
-- **`langchain`**: Chains, agents, and retrieval strategies that make up an application's cognitive architecture.
-- **[LangGraph](https://langchain-ai.github.io/langgraph)**: Build robust and stateful multi-actor applications with LLMs by modeling steps as edges and nodes in a graph. Integrates smoothly with LangChain, but can be used without it.
-- **[LangServe](/docs/langserve)**: Deploy LangChain chains as REST APIs.
-- **[LangSmith](https://docs.smith.langchain.com)**: A developer platform that lets you debug, test, evaluate, and monitor LLM applications.
+- **`langchain-core`**: [Component abstractions](/docs/concepts/#components) and [LangChain Expression Language](/docs/concepts/#langchain-expression-language-lcel).
+- Integration packages (`langchain-openai`, `langchain-anthropic`, `langchain-community`, etc.): Standard interfaces for all the most popular third-party integrations.
+- **`langchain`**: Prebuilt chains, agents, and retrieval strategies for common cognitive architectures.
+- **[`langgraph`](https://langchain-ai.github.io/langgraph)**: Protocol for building robust and stateful multi-actor applications with LLMs by modeling steps as edges and nodes in a graph. Integrates seamlessly with `langchain` and LangSmith, but can be used without them as well.
```

Comment:
protocol plus prebuilt components, but protocol seemed like highest order bit

Reply:
Protocol to me implies a high level description of something rather than an implementation - do we use that term anywhere else?

Why not `framework` or `library` like the LangGraph docs call it?

Reply:
+ 1 to framework for consistency